### ISSUE TITLE
Which file to edit before compiling to set wfi ip as client

### ISSUE THREAD

**Issue by ke-ss** [omersiar/esp-rfid#208]
**What I can do with sucess as follows :**

- I started out believing I could simply flash the latest firmware.bin onto an ESP8266-12E using esptool.py. The ESP8266-12E flashes just fine.
- I can connect to the ESP8266-12E hotspot 192.168.4.1 just fine.
- I can do all this for the three most recent code revisions.
- I managed to install platformio and have used platformio to compile the code from the command line as specified in the esp-rfid github readme file.

**What I have not been able to do :**
 - Get the hotspot to save any of my changes for any version of above firmwares after compiling with platformio and flashing with esptool.py. I thus am not able to get it to become a client with fixed ip 192.168.1.125 on my local wi-fi network, because it simply will not save my changes.
 - I am unable to identify the file to change to set everything up before I compile so that the ESP8266-12E already has a client ip after I have compiled and flashed. I beleive the below is the information I should use for what I want to achieve. I have already placed this information in two file locations before compiling with no avail. **Where is the correct place to place this information before I compile, so that I can power up the ESP8266-12E after it is flashed, with no further configuration necessary?**
 - **Why won't the 192.168.4.1 hotspot save configuration changes that would allow it to become a client?**

config = {
    "command": "configfile",
    "network": {
        "bssid": "70:4F:57:97:52:06",
        "ssid": "datastream5",
        "wmode": 0,
        "hide": 0,
        "pswd": "*******************************",
        "offtime": 0,
        "dhcp": 0,
        "ip": "192.168.1.125",
        "subnet": "255.255.255.0",
        "gateway": "192.168.1.1",
        "dns": "79.79.79.79"
    },
    "hardware": {
        "readerType": 1,
        "wgd0pin": 4,
        "wgd1pin": 5,
        "sspin": 0,
        "rfidgain": 32,
        "rtype": 1,
        "rpin": 4,
        "rtime": 400
    },
    "general": {
        "hostnm": "esp-rfid_door_lock",
        "restart": 0,
        "pswd": "********************************"
    },
    "mqtt": {
        "enabled": 0,
        "host": "192.168.1.120",
        "port": 1883,
        "topic": "front_door_lock",
        "user": "",
        "pswd": ""
    },
    "ntp": {
        "server": "pool.ntp.org",
        "interval": 30,
        "timezone": 0
    }
};

**Comment by omersiar** [comment]
Please use version 0.8.1

**Comment by ke-ss** [comment]
Hi Omer,

This is a great project. 
Thanks to all contributors, yourself included.

I will focus on 0.8.1 from now on.

I have two specific questions about my issues and a few further questions if you don't mind.

 - Which file / location can I put my config in before I compile. 
Where does it go, pls. Do I need to create a new file for this? I tried to place the config in what I though was a couple of obvious locations but it did not set the chip as a client.

 - Any idea why the chip won't write / save to SPIFFS when I save config while connected to 192.168.4.1 hotspot?

Many Weigand readers now come with a pin pad and RFID reader. I hope to utilise both. I think the pin pad additional functionality that's coming will be of interest to very many people.

I would love to learn C and contribute to this project but with other commitments that i already make to community projects contributing here is simply not possible. 

I had a break in at my house recently and two bicycles were stolen. This happened because it is my nature to not lock things. With effective pin pad / rfid locking on my doors I can have the same ease as unlocked but wth the security. It will take me some time to work out how to do it all and implement it.

When I have worked it all out and implemented it I will post up a detailed how to with bill of materials (BOM) somewhere on line which I will link to here. It might take me some time as this is a 'back project'. I intend to give back to the community on this. Using mechanical keys is just so tedious.

Can I also ask, does the for sale chip run flawlessly, and does it run the same code as on this git hub. 

Thanks
